Ginger/GingerCore/Reporter/UserMessagesPool.cs

```diff
@@ -519,10 +519,10 @@ public static void LoadUserMessgaesPool()
             Reporter.UserMessagesPool.Add(eUserMsgKeys.POMCannotDeleteAutoLearnedElement, new UserMessage(eAppReporterMessageType.WARN, "Cannot Delete Auto Leared Element", "The Element you are trying to delete has been leared automatically from page and cannot be deleted", MessageBoxButton.OK,MessageBoxResult.OK));
-            Reporter.UserMessagesPool.Add(eUserMsgKeys.WizardCantMoveNextWhileInProcess, new UserMessage(eAppReporterMessageType.WARN, "Process is still running", "Ginger cannot move next until the process will be finished or stopped" + Environment.NewLine + "please wait the process to be finished or stop it and then click next.", MessageBoxButton.OK, MessageBoxResult.None));
-            Reporter.UserMessagesPool.Add(eUserMsgKeys.WizardCantMovePrevWhileInProcess, new UserMessage(eAppReporterMessageType.WARN, "Process is still running", "Ginger cannot move previous until the process will be finished or stopped" + Environment.NewLine + "please wait the process to be finished or stop it and then click previous.", MessageBoxButton.OK, MessageBoxResult.None));
-            Reporter.UserMessagesPool.Add(eUserMsgKeys.WizardCantFinishWhileInProcess, new UserMessage(eAppReporterMessageType.WARN, "Process is still running", "Ginger cannot move finish wizard until the process will be finished or stopped" + Environment.NewLine + "please wait the process to be finished or stop it and then click finish.", MessageBoxButton.OK, MessageBoxResult.None));
-            Reporter.UserMessagesPool.Add(eUserMsgKeys.AskIfToCloseAgent, new UserMessage(eAppReporterMessageType.QUESTION, "Close Agent?", "To close the Agent '{0}'?", MessageBoxButton.YesNo, MessageBoxResult.No));
+            Reporter.UserMessagesPool.Add(eUserMsgKeys.WizardCantMoveNextWhileInProcess, new UserMessage(eAppReporterMessageType.WARN, "Process is still running", "Ginger cannot move next until the process will be finished or stopped." + Environment.NewLine + "Please wait for the process to be finished or stop it and then click next.", MessageBoxButton.OK, MessageBoxResult.None));
+            Reporter.UserMessagesPool.Add(eUserMsgKeys.WizardCantMovePrevWhileInProcess, new UserMessage(eAppReporterMessageType.WARN, "Process is still running", "Ginger cannot move previous until the process will be finished or stopped." + Environment.NewLine + "Please wait for the process to be finished or stop it and then click previous.", MessageBoxButton.OK, MessageBoxResult.None));
+            Reporter.UserMessagesPool.Add(eUserMsgKeys.WizardCantFinishWhileInProcess, new UserMessage(eAppReporterMessageType.WARN, "Process is still running", "Ginger cannot move finish wizard until the process will be finished or stopped." + Environment.NewLine + "Please wait for the process to be finished or stop it and then click finish.", MessageBoxButton.OK, MessageBoxResult.None));
```

Comment:
1. why 3 same UserMessages? make the next/Previous/finish word as parameter {0} and then set it in calling function based on scenario 
2. the header should be with capital letters like: "Process is Still Running"
3. no need to say "Ginger..." just: "Move Next operation is blocked till process will be finished..."